jiant/tasks/lm.py

```diff
@@ -172,3 +178,102 @@ def __init__(self, path, *args, **kw):
             "val": os.path.join(path, "valid.sentences.txt"),
             "test": os.path.join(path, "test.sentences.txt"),
         }
+
+
+@register_task("mlm", rel_path="WikiText103/")
+class MaskedLanguageModelingTask(Task):
```

Comment:
It shouldn't be possible to resolve the reference to `Task` without importing it, or am I missing something?